spec/datadog/di/integration/instrumentation_spec.rb

```diff
@@ -572,7 +572,8 @@ def run_test
                 expect(status).to match(expected_emitting_payload)
               end
               expect(component.probe_notifier_worker).to receive(:add_snapshot) do |snapshot|
-                expect(snapshot.fetch(:message)).to eq 'hello '
+                # TODO should we serialize nil as empty string?
```

Comment:
an explicit "nil" string, as it's implemented now seems ok, given it looks like the pattern we are implementing is more similar to pp than print.